homestar-runtime/src/network/swarm.rs

```diff
@@ -65,6 +65,28 @@ pub(crate) async fn new(settings: &settings::Node) -> Result<Swarm<ComposedBehav
     Ok(swarm)
 }
 
+fn swarm_startup<T: NetworkBehaviour>(
```

Comment:
minor: for consistency, since this is already in the swarm module, just `startup` should work. We'll essentially make `startup` a definition that's for any process that spins up via spawn.